statsmodels/tools/testing.py

```diff
@@ -16,8 +16,10 @@ def strip_rc(version):
 
 
 def is_pandas_min_version(min_version):
-    '''check whether pandas is at least min_version '''
-    return StrictVersion((pandas.__version__[:6])) >= min_version
+    '''check whether pandas is at least min_version
+    '''
+    from pandas import __version__ as pversion
+    return StrictVersion(strip_rc(pversion)) >= min_version
```

Comment:
not sure I see the problem
you seem to be jumping thru hoops to do simple version checking

we do this all the time (meaning using Loose/Strict Version) for lots of packages and it works fine